server/access/rest.go

```diff
@@ -78,8 +81,12 @@ func NewRestServer(logger *zerolog.Logger, adapter *adapters.AccessAdapter, chai
 	if debug {
 		debugLogger = zerolog.New(os.Stdout)
 	}
+	var restCollector module.RestMetrics = metrics.NewNoopCollector()
 
-	restCollector := metrics.NewNoopCollector()
+	// only collect metrics if not test
```

Comment:
When would that be the case? When running the tests of the emulator?

Reply:
yeah emulator tests run in parallel, but prometheus wants single metric instance. So we if we have two REST server instance, it is complaining.